groups/terraform.auto.tfvars

```diff
@@ -0,0 +1,3 @@
+team_x_dba_members = [
```

Comment:
If we wanted to make things truly foolproof, we could have multiple *.auto.tfvars files, as the number of groups grows.